cmd/quickstart/uninstall.go

```diff
@@ -105,15 +113,69 @@ func (o *uninstallOptions) uninstallOCIRegistry(ctx context.Context, k8sClient c
 	return ociRegistry.uninstall(ctx)
 }
 
-func (o *uninstallOptions) uninstallLandscaper(ctx context.Context) error {
+func (o *uninstallOptions) uninstallLandscaper(ctx context.Context, k8sClient client.Client) error {
+	fmt.Println("Removing deployer registrations")
+	deployerRegistrations := v1alpha1.DeployerRegistrationList{}
+	if err := k8sClient.List(ctx, &deployerRegistrations); err != nil {
+		return err
+	}
+
+	for _, registration := range deployerRegistrations.Items {
+		if err := k8sClient.Delete(ctx, &registration); err != nil {
+			return err
+		}
+	}
+
+	if err := waitForRegistrationsRemoved(ctx, k8sClient); err != nil {
+		return err
+	}
+
+	fmt.Println("helm delete")
```

Comment:
Think this is not needed as the "helm delete" command will already be logged to console in util. ExecCommandBlocking() from the next line

Reply:
I have removed the line